.github/ISSUE_TEMPLATE/functional_bug.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Functional Bug
+about: Use this template to report functionally incorrect or misleading documentation.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Summary
+
+Replace this with a short summary (one sentence or short paragraph) of
+the issue you are describing.
+
+## Affected paths
+
+List the URL(s) where you found faulty behavior. You can use URLs from
```

Comment:
The way I see it, "faulty behavior" should be replaced by "inaccurate information". (We are not concerned with anyone's behavior, but we do care about the quality/accuracy of the provided information.)

Reply:
Good suggestion.